src/MenuGraphr.ts

```diff
@@ -994,7 +994,7 @@ export class MenuGraphr implements IMenuGraphr {
         }
 
         // If the next word would pass the edge of the menu, move down a line
-        if (x + this.computeFutureWordLength(words[i + 1], textWidth, textPaddingX) >= menu.right - menu.textXOffset) {
+        if (x + this.computeFutureWordLength(words[i + 1], textWidth, textPaddingX) >= menu.right - menu.textXOffset - this.GameStarter.unitsize * 2) {
```

Comment:
>  this.GameStarter.unitsize * 2

This should be a configurable amount. Not every menu will want it to be `unitsize * 2`.